BlockParty.py

```diff
@@ -553,7 +558,18 @@ def draw_escape(self, cairo_ctx):
 
         self.draw_string(
             cairo_ctx, 'Press ESC to exit',
-            self.xnext + self.bwpx * 2.5 + 60, self.window_h - 50, True)
+            self.xnext + self.bwpx * 2.5, self.window_h - 50, True)
+        cairo_ctx.fill()
+
+    def draw_back(self, cairo_ctx):
+        cairo_ctx.set_line_width(1)
+        cairo_ctx.set_source_rgb(0,
+                                 0,
+                                 0)
```

Comment:
I think this can all be on the same line as it currently hinders readability.

Reply:
I agree.  And use black.

Reply:
Done it in 163e3a5. Used the already used style to pass black color.